src/Sylius/Bundle/AdminBundle/Resources/views/Product/_mainImage.html.twig

```diff
@@ -5,5 +5,8 @@
 {% else %}
     {% set path = '//placehold.it/50x50' %}
 {% endif %}
-
-<img src="{{ path }}" alt="" class="ui bordered image" />
+{% if product.images.first.path|split('.')|last is same as 'svg' %}
+    <img src="{{ path }}" alt="" class="ui bordered image" style="max-height: 100px; max-width: 150px" />
+{% else %}
+    <img src="{{ path }}" alt="" class="ui bordered image" />
+{% endif %}
```

Comment:
Instead of this `if`, you could maybe change css classes for grids and other places with uploaded images

Reply:
I checked all places with images and if i am right this grid is the only one without proper sizing provide by semanticUI